native/explorer/src/series.rs

```diff
@@ -548,7 +548,52 @@ pub fn s_in(s: ExSeries, rhs: ExSeries) -> Result<ExSeries, ExplorerError> {
         DataType::Date => s.date()?.is_in(&rhs)?,
         DataType::Time => s.time()?.is_in(&rhs)?,
         DataType::Datetime(_, _) => s.datetime()?.is_in(&rhs)?,
-        dt => panic!("is_in/2 not implemented for {dt:?}"),
+        DataType::Categorical(Some(mapping)) => {
+            let l_logical = s.categorical()?.logical();
+
+            match rhs.dtype() {
+                DataType::Utf8 => {
+                    let mut r_ids: Vec<Option<u32>> = vec![];
```

Comment:
Are we sure we should implement this? If Polars don't implement it, then maybe we should not? Or we should ask it to be implemented there.